// [[Reason: checker.typeToTypeNode deliberately fails on types that originate from node_modules]] ////

//// [tests/cases/compiler/moduleResolutionWithSuffixes_one_externalTSModule.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,20 +1,4 @@
 
 
 //// [/bin/test.d.ts]
 export {};
-
-/// [Errors] ////
-
-/node_modules/some-library/index.ios.ts(1,17): error TS9007: Function must have an explicit return type annotation with --isolatedDeclarations
-
-
-==== /test.ts (0 errors) ====
-    import { ios } from "some-library";
-    
-==== /node_modules/some-library/index.ios.ts (1 errors) ====
-    export function ios() {}
-                    ~~~
-!!! error TS9007: Function must have an explicit return type annotation with --isolatedDeclarations
-!!! related TS9031 /node_modules/some-library/index.ios.ts:1:17: Add a return type to the function declaration
-==== /node_modules/some-library/index.ts (0 errors) ====
-    export function base() {}
\ No newline at end of file
